src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/AccountController.java

```diff
@@ -337,4 +337,15 @@ public ResponseEntity<? extends PiUser> createSystemAdminAccount(
         return ResponseEntity.ok(systemAdminAccountService.addSystemAdminAccount(account, issuerId));
     }
 
+    @ApiResponses({
+        @ApiResponse(responseCode = OK_CODE, description = "System Admin Information"),
+        @ApiResponse(responseCode = AUTH_ERROR_CODE, description = NOT_AUTHORIZED_MESSAGE),
+        @ApiResponse(responseCode = NOT_FOUND_ERROR_CODE, description = "User not found")
+    })
+    @Operation(summary = "Get System Admin info")
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping("/get-info/{issuerId}")
```

Comment:
I think this should be generic as theres no restriction for System Admin.

Something like GET /azure would work

I'd also suggest the issuerID is changed to provenanceId to keep in generic

Reply:
As discussed, it's fixed.

Reply:
Cheers. Text all looks OK - however the endpoint name should still be changed.

I think it should be /azure/{provenanceUserId} would work?

Reply:
fixed.